app/assets/v2/js/pages/bounty_share.js

```diff
@@ -21,7 +21,7 @@ $('.modal-link').click(function(e) {
 const sendInvites = (users) => {
   let usersId = [];
   let msg = $('#shareText').val();
-  const url = document.issueURL;
+  let bountyId = document.result.pk;
```

Comment:
Doesn't look like bountyId gets updated within this ! 
should we just leave it as const ? 

Also should we be extra safe and do `const bountyId = document.result && document.result.pk` ?